sdk/ai/azure-ai-generative/azure/ai/generative/index/__init__.py

```diff
@@ -6,17 +6,7 @@
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)
 
 from ._build_mlindex import build_index
-from ._functions import (
```

Comment:
Where are these functions moved to? This will be a breaking change.

Reply:
I deleted these functions since there are not used in this repo, and _functions should be a private module.

Reply:
As we discussed offline, these functions should be kept